### DISCUSSION TITLE
Scaling down a wide table for PDF

### DISCUSSION THREAD

**Discussion by dhicks** [quarto-dev/quarto-cli#1354] [Q&A] [answered]
I'm working on an article manuscript that includes several very wide tables (regression coefficients from 3+ models) built using `gtsummary`.  I'm having trouble figuring out a good way to display them in a PDF document.   

[MWE materials](https://github.com/quarto-dev/quarto-cli/files/9086411/mwe.zip)

Here's what I've tried: 

- `knitr`'s columns are too narrow; I guess it kind of looks okay in landscape (table 4). 
- `huxtable` either has narrow columns or goes off the page. 
- Including a PDF of the table alone (generated using `webshot`) looks great (page 12), but cross-references don't work. 
- I can't use [`\adjustbox`](https://tex.stackexchange.com/questions/97505/shrink-table-to-fit-on-a-page-or-keep-it-as-it-is/97506#97506) because it needs to go between the `table` and `tabular` environments.

**Comment by arnaudgallou** [comment, marked as answer]
You can try `kableExtra::kable_styling(latex_options = "scale_down")` to adjust the width of the table to the page. `kableExtra` also allows you to control column widths and font size. That and possibly landscape mode should cover all cases. If not, it probably means that you need to rethink the structure of your table. E.g., one column = one variable or split your tables in smaller ones.

**Comment by produnis** [comment]
And how does it work with typst?

**Reply by mcanouil** [reply]
It does not as `kableExtra` does not support Typst and does not allow HTML table for non-HTML formats.

Side note: it's usually preferable to open a new thread that reference a 3-years old one rather than resurrecting it.
